src/store/index.js

```diff
@@ -51,17 +55,41 @@ export default createStore({
         address: 4,
         completed: false
       }
-    ]
+    ],
+    todos: [],
+    factory: {},
+    accountNum: null
   },
   getters: {
     /* eslint-disable-next-line */
-    getSingleCampaign: state => address => state.campaigns.find((campaign) => campaign.address === address)
+    getSingleCampaign: state => address => state.campaigns.find((campaign) => campaign.address === address), 
+    getFactory: (state) => state.factory
   },
   mutations: {
     ADD_CAMPAIGN: (state, payload) => {
       state.campaigns = [...state.campaigns, payload];
+    },
+    setTodos: (state, todos) => (state.todos = todos),
+    setFactory: (state, instance) => (state.factory = instance),
+    setAccountNum: (state, accNum) => (state.accountNum = accNum)
+  },
+  actions: {
+    fetchTodos: async ({ commit }) => {
+      const response = await axios.get(
+        'https://jsonplaceholder.typicode.com/todos'
+      );
+
+      commit('setTodos', response.data);
+    },
+    fetchFactory: async ({ commit }) => {
+      const instance = await factory.at('0x7c70286f6991c660a0cC6d52A74aEBbDE45Da380')
+      commit('setFactory', instance);
+    },
+    fetchAccountNum: async ({ commit }) => {
+      const accounts = await web3.eth.getAccounts()
+      commit('setAccountNum', accounts[0]);
```

Comment:
So if this is the address for the rinkby network blockchain, it should still work in production?